cli/config.py

```diff
@@ -0,0 +1,62 @@
+"""
+Provide implementation of the Remme Core CLI checking configurations manipulating.
```

Comment:
`Provide implementation of the Remme Core CLI checking configurations manipulating` > `Provide implementation of the CLI configurations file`.

Reply:
@dmytrostriletskyi, done.